core/runtime/pom.xml

```diff
@@ -213,7 +213,14 @@
 
                         <!-- Testcontainers support.-->
                         <parentFirstArtifact>org.testcontainers:testcontainers</parentFirstArtifact>
+                        <parentFirstArtifact>com.github.docker-java:docker-java</parentFirstArtifact>
                         <parentFirstArtifact>com.github.docker-java:docker-java-api</parentFirstArtifact>
+                        <parentFirstArtifact>com.github.docker-java:docker-java-transport</parentFirstArtifact>
+                        <parentFirstArtifact>com.github.docker-java:docker-java-transport-zerodep</parentFirstArtifact>
+                        <parentFirstArtifact>junit:junit</parentFirstArtifact>
+                        <parentFirstArtifact>com.fasterxml.jackson.core:jackson-annotations</parentFirstArtifact>
```

Comment:
We definitely don't want this

Reply:
So, should I remove all dependencies?

Reply:
![image](https://user-images.githubusercontent.com/24874651/168335423-ce9301ed-48b5-4bbd-a310-59976a2f6d2b.png)

Reply:
I got this when I remove that dependency.

Reply:
Perhaps it's important to mention, I am using podman.